File: 231226/좀 더 어려운 수학 점수/math-scores-are-more-difficult.py
a_score = input().split()
a_math = int(a_score[0])
a_eng = int(a_score[1])

b_score = input().split()
b_math = int(b_score[0])
b_eng = int(b_score[1])

if a_math > b_math or (a_math == b_math and a_eng > b_eng):
    print("A")
else:
    print("B")

File: 240201/좀 더 어려운 수학 점수/math-scores-are-more-difficult.js
const fs = require("fs");
let score = fs.readFileSync(0).toString().trim().split("\n");

let a = score[0].split(" ");
let b = score[1].split(" ");

let aMath = Number(a[0]), aEng = Number(a[1]);
let bMath = Number(b[0]), bEng = Number(b[1]);

if (aMath > bMath) {
    console.log("A");
} else if(bMath > aMath) {
    console.log("B");
} else {
    console.log(aEng > bEng ? "A" : "B");
}